src/applications/mhv/secure-messaging/tests/e2e/triage-team-tests/secure-messaging-thread-no-association-at-all.cypress.spec.js

```diff
@@ -0,0 +1,116 @@
+
+describe('Verify thread - No association at all', () => {
+  const site = new SecureMessagingSite();
+  const landingPage = new PatientInboxPage();
+  beforeEach(() => {
+    site.login();
+  });
+
+  it('landing page view', () => {
+    secureMessagingLandingPage.loadMainPage(mockNoRecipients);
+    cy.injectAxe();
+    cy.axeCheck(AXE_CONTEXT, {
+      rules: {
+        'aria-required-children': {
+          enabled: false,
+        },
+      },
+    });
+
+    cy.get(Locators.LINKS.CREATE_NEW_MESSAGE).should('not.exist');
+  });
+
+  it('inbox with messages page view', () => {
+    landingPage.loadInboxMessages(
+      mockMessages,
+      mockSingleMessage,
+      mockNoRecipients,
+    );
+    cy.injectAxe();
+    cy.axeCheck(AXE_CONTEXT, {
+      rules: {
+        'aria-required-children': {
+          enabled: false,
+        },
+      },
+    });
+
+    cy.get(Locators.LINKS.CREATE_NEW_MESSAGE).should('not.exist');
+    cy.get(Locators.LINKS.GO_TO_INBOX).should('not.exist');
+
+    cy.get(Locators.ALERTS.BLOCKED_GROUP)
+      .find('h2')
+      .should('have.text', Alerts.NO_ASSOCIATION.AT_ALL_HEADER);
+
+    cy.get(Locators.ALERTS.BLOCKED_GROUP)
+      .find('p')
+      .should('have.text', Alerts.NO_ASSOCIATION.PARAGRAPH);
+
+    cy.get(Locators.ALERTS.BLOCKED_GROUP)
+      .find('a')
+      .should('have.text', Alerts.NO_ASSOCIATION.LINK);
+
+    cy.get(Locators.ALERTS.BLOCKED_GROUP)
+      .find('a')
+      .should('have.attr', 'href', '/find-locations/');
+  });
+
+  it.skip('detailed view', () => {
+    landingPage.loadInboxMessages(
+      mockMessages,
+      mockSingleMessage,
+      mockNoRecipients,
+    );
+    // landingPage.loadSingleThread();
+
+    cy.injectAxe();
+    cy.axeCheck(AXE_CONTEXT, {
+      rules: {
+        'aria-required-children': {
+          enabled: false,
+        },
+      },
+    });
+
+    cy.get(Locators.ALERTS.BLOCKED_GROUP)
+      .should('be.visible')
+      .and('include.text', Alerts.NO_ASSOCIATION.AT_ALL_HEADER);
+
+    // cy.get(Locators.ALERTS.BLOCKED_GROUP)
+    //   .shadow()
+    //   .find('#alert-body')
+    //   .should('have.class', 'closed');
+
+    // cy.get(Locators.ALERTS.BLOCKED_GROUP).click({
+    //   waitForAnimations: true,
+    // });
+    //
+    // cy.get(Locators.ALERTS.BLOCKED_GROUP)
+    //   .shadow()
+    //   .find('#alert-body')
+    //   .should('have.class', 'open');
+    //
+    // cy.get(Locators.ALERTS.BLOCKED_GROUP)
+    //   .find('a')
+    //   .should('have.attr', 'href', '/find-locations/');
+    //
+    // cy.get(Locators.BUTTONS.REPLY).should('not.exist');
+    //
+    // // TODO move these assertion up after alert text fixing
+    //
+    // cy.get(Locators.ALERTS.BLOCKED_GROUP)
+    //   .find('p')
+    //   .should('have.text', Alerts.NO_ASSOCIATION.PARAGRAPH);
+    //
+    // cy.get(Locators.ALERTS.BLOCKED_GROUP)
+    //   .find('a')
+    //   .should('have.text', Alerts.NO_ASSOCIATION.LINK);
```

Comment:
@fazilqa @haniasma do you guys need this commented code? If not, it is probably better to clean it up

Reply:
fixed and cleaned up